### ISSUE TITLE
Unable to run wappalyzer because of Type Error: wappalyzer.open(...).analyze is not a function

### ISSUE THREAD

**Issue by kausality** [wappalyzer/wappalyzer#7566]
**Platform**
Which platform does the bug apply to?
* NPM

**Describe the bug**
I am using the example from ReadMe.
```
const Wappalyzer = require('wappalyzer');

const urls = process.argv.slice(2)

const wappalyzer = new Wappalyzer()

;(async function() {
  try {
    await wappalyzer.init()

    const results = await Promise.all(
      urls.map(async (url) => ({
        url,
        results: await wappalyzer.open(url).analyze()
      }))
    )

    console.log(JSON.stringify(results, null, 2))
  } catch (error) {
    console.error(error)
  }

  await wappalyzer.destroy()
})()
```

Result:
```
node run_wpz.js https://example.com
TypeError: wappalyzer.open(...).analyze is not a function
    at /Users/user1/wpz_dep/wappalyzer_code/run_wpz.js:14:45
    at Array.map (<anonymous>)
    at /Users/user1/wpz_dep/wappalyzer_code/run_wpz.js:12:12
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

```
**Additional context**
mac OS 13.2.1 - M1

**Comment by homeboy445** [comment]
Hi, I think there's a problem in your code which is that you're not awaiting the **open** method (which returns a promise) and directly calling the **analyze** method, which is resulting in this error, here's the fixed code:

```
const Wappalyzer = require('wappalyzer');

const urls = process.argv.slice(2)

const wappalyzer = new Wappalyzer()

;(async function() {
  try {
    await wappalyzer.init()

    const results = await Promise.all(
      urls.map(async (url) => ({
        url,
        results: await (await wappalyzer.open(url)).analyze()
      }))
    )

    console.log(JSON.stringify(results, null, 2))
  } catch (error) {
    console.error(error)
  }

  await wappalyzer.destroy()
})()

```

**Comment by kausality** [comment]
@homeboy445 thank you for the input. This works. 
I think wappalyzer README needs to be updated at: [](https://github.com/wappalyzer/wappalyzer/blob/master/src/drivers/npm/README.md)

I took it from the section where there is an example of Multiple URL's processing. Looks like it is outdated.

**Comment by AliasIO** [comment]
Fixed in 921e725ab92f3df188cf2c97ab762fa22d38187f. 👍
